src/php/ext/grpc/channel.c

```diff
@@ -615,12 +622,11 @@ static void php_grpc_target_bound_dtor(php_grpc_zend_resource *rsrc
 #ifdef GRPC_PHP_DEBUG
 
 /**
-* Clean all channels in the persistent. Test only.
+* Clean all channels in the persistent
 * @return void
 */
 PHP_METHOD(Channel, cleanPersistentList) {
-  zend_hash_clean(&grpc_persistent_list);
```

Comment:
We can probably leave this alone. Or we should see whether we need to call `zend_hash_destroy` like the other places too.

Reply:
comment addressed